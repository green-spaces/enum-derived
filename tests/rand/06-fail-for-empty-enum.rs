use enum_derived::Rand;

#[derive(Rand, PartialEq, Eq, Hash, Debug)]
pub enum VariantFree {}

fn main() {}
