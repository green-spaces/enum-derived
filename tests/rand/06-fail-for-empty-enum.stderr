error: proc-macro derive panicked
 --> tests/rand/06-fail-for-empty-enum.rs:3:10
  |
3 | #[derive(Rand)]
  |          ^^^^
  |
  = help: message: Enum must have at least one variant defined
